rename_ref_data.py

```diff
@@ -0,0 +1,21 @@
+import hail as hl
+
+ht = hl.read_table('gs://seqr-reference-data/GRCh37/all_reference_data/combined_reference_data_grch37.ht')
+ht = ht.drop('g1k')
+ht.transmute(
+    gnomad_genome_coverage=hl.Struct(x10=ht.gnomad_genome_coverage),
+    gnomad_exome_coverage=hl.Struct(x10=ht.gnomad_exome_coverage),
+)
+ht.annotate_globals(datasets=ht.datasets.remove('1kg'))
+ht.write('gs://seqr-reference-data/GRCh37/all_reference_data/combined_reference_data_grch37.ht', overwrite=True)
```

Comment:
hail wont let you write to the same location so this would need to write to a temp location, maybe checkpoint it, and then you could write back to the original location.